server/gorillamux/problems/problems.go

```diff
@@ -0,0 +1,47 @@
+package problems
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/SKF/go-rest-utility/problems"
+)
+
+type NotFoundProblem struct {
+	problems.BasicProblem
+}
+
+func NotFound() NotFoundProblem {
+	return NotFoundProblem{
+		BasicProblem: problems.BasicProblem{
+			Type:   "/problems/route-not-found",
+			Title:  "The requested endpoint could not be found.",
+			Status: http.StatusNotFound,
+			Detail: "Ensure that the URI is a valid endpoint for the service.",
+		},
+	}
+}
+
+type MethodNotAllowedProblem struct {
+	problems.BasicProblem
+	Method  string   `json:"requested_method,omitempty"`
```

Comment:
I tried to lobby for us to use snake_case, but it was decided we should use camelCase. I think that's why I used allowed and method instead.